README.md

```diff
@@ -33,7 +33,8 @@ It checks `.env` files for problems that may cause the application to malfunctio
 &nbsp;&nbsp;&nbsp;&nbsp;✅<a href="#leading-character">Leading character</a><br />
 &nbsp;&nbsp;&nbsp;&nbsp;✅<a href="#lowercase-key">Lowercase key</a><br />
 &nbsp;&nbsp;&nbsp;&nbsp;✅<a href="#space-character">Space character</a><br />
-&nbsp;&nbsp;&nbsp;&nbsp;✅<a href="#unordered-Key">Unordered Key</a>
+&nbsp;&nbsp;&nbsp;&nbsp;✅<a href="#unordered-Key">Unordered Key</a><br />
+&nbsp;&nbsp;&nbsp;&nbsp;✅<a href="#quote-character">Quote character</a>
```

Comment:
```suggestion
&nbsp;&nbsp;&nbsp;&nbsp;✅<a href="#quote-character">Quote character</a> (Will be available in v2.0.0)
```
Also, please add the additional information that this check will be available in v2.0.0.